lib/api/controllers/account_controller.ex

```diff
@@ -1,18 +1,79 @@
 defmodule NabooAPI.AccountController do
   use Phoenix.Controller, namespace: NabooAPI, root: "lib/api"
+  use PhoenixSwagger
 
   alias Naboo.Accounts
   alias Naboo.Accounts.Account
   alias NabooAPI.AccountView
   alias NabooAPI.Views.Errors
 
+  swagger_path(:index) do
+    get("api/account")
+    summary("Lists users")
+    description("Lists all users in the database")
+    produces("application/json")
+    deprecated(false)
+
+    response(200, "index.json", %{},
+      example: %{
+        accounts: [
+          %{
+            email: "test@test.com",
+            encrypted_password: "sqdqsd",
+            password: "test_t",
+            is_admin: false,
+            name: "test"
+          },
+          %{
+            email: "test2@test.com",
+            encrypted_password: "sqdqsd58",
+            password: "tes",
+            is_admin: false,
+            name: "test2"
+          }
+        ]
+      }
+    )
+  end
+
   def index(conn, _params) do
     conn
     |> put_view(AccountView)
     |> put_status(:ok)
     |> render("index.json", accounts: Accounts.list_accounts())
   end
 
+  swagger_path(:create) do
+    post("/api/account")
+    summary("Create an user")
+    description("Create an user in the database")
+    produces("application/json")
+    deprecated(false)
+
+    parameter(:account_params, :body, :Account, "informations of the account",
+      required: true,
+      example: %{
+        account: %{
+          email: "test@test.com",
+          password: "test_t",
+          name: "test"
+        }
+      }
+    )
+
+    response(200, "show.json", %{},
+      example: %{
+        account: %{
+          email: "test@test.com",
+          encrypted_password: "sqdqsd",
+          password: "test_t",
```

Comment:
Same here -> We don't serialize this (`password` is never even persisted into database)